tutorial/developing_packages/workspaces.rst

```diff
@@ -0,0 +1,285 @@
+
+      def generate(self):
+         deps = CMakeDeps(self)
+         deps.generate()
+         tc = CMakeToolchain(self)
+         tc.generate()
+
+      def layout(self):
+         cmake_layout(self)
+
+   class Ws(Workspace):
+      def root_conanfile(self):
+         return MyWs  # Note this is the class name
+
+
+The role of the ``class MyWs(ConanFile)`` embedded conanfile is important, it defines
+the super-project necessary generators and layout.
+
+The ``conan workspace install`` does not install the different editables separately, for
+this command, the editables do not exist, they are just treated as a single "node" in
+the dependency graph, as they will be part of the super-project build. So there is only
+a single generated ``conan_toolchain.cmake`` and a single common set of dependencies
+``xxx-config.cmake`` files for all super-project external dependencies.
+
+
+The template above worked without external dependencies, but everything would work
+the same when there are external dependencies. This can be tested with:
+
+.. code-block:: bash
+
+   $ conan new cmake_lib -d name=mymath
+   $ conan create .
+   $ conan new workspace -d requires=mymath/0.1
+   $ conan workspace install
+   $ cmake ...
+
+
+.. note::
+
+   The current ``conan new workspace`` generates a CMake based super project.
+   But it is possible to define a super-project using other build systems, like a
+   MSBuild solution file that adds the different ``.vcxproj`` subprojects. As long as
+   the super-project knows how to aggregate and manage the sub-projects, this is possible.
+
+   It might also be possible for the ``add()`` method in the ``conanws.py`` to manage the
+   addition of the subprojects to the super-project, if there is some structure.
+
+
+.. _tutorial_workspaces_orchestrated:
+
+Orchestrated build
+------------------
+
+Conan workspaces can also build the different ``packages`` separately, and taking into account if there are ``products``
+consuming them.
+
+Let's create another structure to understand better how it works. Now, let's create it from scratch instead of using
+the conan-new-command template:
+
+.. code-block:: bash
+
+   $ conan new cmake_lib -d name=hello -d version=1.0 -o hello
+   $ conan new cmake_exe -d name=app -d version=1.0 -d requires=hello/1.0 -o app
+   $ conan workspace init mywksp
+
+Those commands created a file structure like this:
+
+.. code-block:: text
+
+   .
+   ├── app
+   │    ├── CMakeLists.txt
+   │    ├── conanfile.py
+   │    ├── src
+   │    │    ├── app.cpp
+   │    │    ├── app.h
+   │    │    └── main.cpp
+   │    └── test_package
+   │        └── conanfile.py
+   ├── hello
+   │    ├── CMakeLists.txt
+   │    ├── conanfile.py
+   │    ├── include
+   │    │    └── hello.h
+   │    ├── src
+   │    │    └── hello.cpp
+   │    └── test_package
+   │        ├── CMakeLists.txt
+   │        ├── conanfile.py
+   │        └── src
+   │            └── example.cpp
+   └── mywksp
```

Comment:
The orchestrated build highly benefits from having the natural workspace definition in the root, or using the new ``conanws`` folder feature, but better it be automatic than using a custom ``mywks`` folder that needs moving into the folder all the time.
If having the files in the root, the commands like ``add`` are also simplified to ``conan workspace add hello`` from the root too.